src/device/r4300/interrupt.c

```diff
@@ -290,13 +293,15 @@ int save_eventqueue_infos(const struct cp0* cp0, char *buf)
 void load_eventqueue_infos(struct cp0* cp0, const char *buf)
 {
     int len = 0;
+    uint64_t* cp0_count = r4300_cp0_get_count(cp0);
 
     clear_queue(&cp0->q);
 
     while (*((const unsigned int*)&buf[len]) != 0xFFFFFFFF)
     {
         int type = *((const unsigned int*)&buf[len]);
-        unsigned int count = *((const unsigned int*)&buf[len+4]);
```

Comment:
type == SPECIAL_INT shouldn't not be added to the event list

Reply:
We should probably also need to re-schedule the COMPARE_INT here

Reply:
I'm not sure what you mean by re-schedule COMPARE_INT, won't it be in the event queue?